### PR TITLE
Feature curriculum-content

### PR DESCRIPTION
**Pull request by VHDLNER**
-added feature to fetch contents and module by curriculum id 
-solved some issue with previous feature "ordering content "
- solved conflict  with dev

### CODE REVIEW

**Comment by carolynrehm** [comment]
There are a few files here (scripts/CMSForceSQL.sql, src/main/java/com/revature/controllers/CurriculumController.java, and a couple others) that don't seem to need to be in this pull request

**Comment by carolynrehm** [comment]
Also you'll need to pull from dev

**Comment by VHDLNER** [comment]
I have pulled from dev, i believe now there will no conflict.
also removed those files you mentioned

File: src/main/java/com/revature/services/LinkService.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import java.util.List;
 import java.util.Set;
 
+import com.netflix.ribbon.proxy.annotation.Content;
```
Comment (carolynrehm) [inline src/main/java/com/revature/services/LinkService.java:7]: is this necessary?

File: src/main/java/com/revature/controllers/ModuleController.java [hunk 2]
```diff
@@ -87,6 +85,11 @@
         return ResponseEntity.ok(jsonModules);
     }
 	
+//	// return list of  modules attached to a given curriculum
+//	@GetMapping("/{id}/curriculum")
+//    public ResponseEntity<Set<Module>> getModuleByCurriculumId(@PathVariable int id) {
+//		return ResponseEntity.ok(moduleService.getModulesByCurricumId(id));
+//	}
```
Comment (carolynrehm) [inline src/main/java/com/revature/controllers/ModuleController.java:92]: why include this?

**Review by carolynrehm — CHANGES_REQUESTED** [review]
All of these files don't seem to be necessary:
src/test/java/com/revature/servicestests/ModuleServiceTest.java
src/test/java/com/revature/controllertests/ModuleControllerTest.java
src/test/java/com/revature/controllertests/LinkControllerTest.java
src/main/java/com/revature/services/ModuleServiceImpl.java
src/main/java/com/revature/services/ModuleService.java
src/main/java/com/revature/services/LinkServiceImpl.java
src/main/java/com/revature/services/CurriculumServiceImpl.java
src/main/java/com/revature/services/LinkService.java
src/main/java/com/revature/services/CurriculumModuleServiceImpl.java
src/main/java/com/revature/services/CurriculumModuleService.java
src/main/java/com/revature/repositories/LinkRepository.java
src/main/java/com/revature/repositories/CurriculumRepository.java
src/main/java/com/revature/repositories/CurriculumModuleRepository.java
src/main/java/com/revature/repositories/ContentRepository.java
src/main/java/com/revature/entities/CurriculumModule.java
src/main/java/com/revature/entities/Curriculum.java

Also, the files that were removed just needed to be removed from the pull request, not the entire code base (e.g. we still want the sql script)

There also seems to be some commented out methods that don't seem necessary

File: src/main/java/com/revature/controllers/ModuleController.java (hunk 2 above)
Comment (VHDLNER) [reply src/main/java/com/revature/controllers/ModuleController.java:92]: I include it before I implemented another method on the content controller that gets the modules and content by the curriculum. id it should be removed now.

File: src/main/java/com/revature/services/LinkService.java (hunk 1 above)
Comment (VHDLNER) [reply src/main/java/com/revature/services/LinkService.java:7]: no, i will remove it

**Review by VHDLNER — COMMENTED** [review]
i removed them from pull request

**Comment by carolynrehm** [comment]
The files need to be consistent with the dev branch, not removed from the code base entirely.
There are also remaining merge conflicts.

File: src/main/java/com/revature/controllers/ContentController.java [hunk 3]
```diff
@@ -92,6 +92,13 @@
 	public ResponseEntity<Set<Link>> getLinksByContentId(@PathVariable int id) {
 		return ResponseEntity.ok(contentService.getLinksByContentId(id));
 	}
+	
+	// return list of  links attached to a given curriculum
+	@GetMapping("/{id}/curriculum")
```
Comment (carolynrehm) [inline src/main/java/com/revature/controllers/ContentController.java:97]: the correct mapping for this resource would actually be curriculum/{id}/links

Comment (VHDLNER) [reply src/main/java/com/revature/controllers/ContentController.java:97]: i was trying to be consistent with the previous method used to get links but i will change it
- i  will make test for that method and solve conflict and then push it again

**Comment by carolynrehm** [comment]
moved to #233

**Pull request closed without merging**